learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+# Migrating from Algolia to Meilisearch
+
+This guide will walk you through exporting your Algolia data and indexing it in Meilisearch using a Node.js script. To skip directly to the finished script, [click here](#finished-script).
+
+This guide also includes comparisons of Meilisearch and Algolia's [APIs](#apis), [index settings, parameters](#settings-and-parameters), and [support for the `instantsearch.js` library](#front-end-components).
+
+This is a practical guide for current Algolia users making the transition to Meilisearch. We also provide [a high-level comparison of the two search companies and their products](/learn/what_is_meilisearch/comparison_to_alternatives.md#meilisearch-vs-algolia) elsewhere.
+
+## Migration script
+
+Migrating indexes from Algolia to Meilisearch is a straightforward process:
+
+1. Export your data stored in Algolia
+2. Upload your data to Meilisearch
+3. (optional) Configure your settings to approximate the settings of your Algolia index
+
+This guide uses a [Node.js](https://nodejs.org/en/) script to upload Algolia index data to Meilisearch. If you would like to write this script in another language, we provide [SDKs for most common languages](/learn/what_is_meilisearch/sdks.md) and [welcome contributions to this guide](https://github.com/meilisearch/documentation/edit/master/learn/getting_started/algolia_migration.md).
+
+Before continuing, make sure you have both Meilisearch and Node.js installed, and access to a command-line terminal. If you're unsure, see our [quick start](/learn/getting_started/quick_start.md) for instructions on installing Meilisearch.
```

Comment:
To be safe, we can mention the current node version as well as the versions of meilisearch and algoliasearch dependencies that we are using.

Node v16.14
algoliasearch 4.13
meilisearch  0.25

Reply:
It is always best to mention the exact versions to help the user debug and also to avoid having obsolete guides. With the specified versions we guarantee to the user that this guide works in these circumstances. 

I think we should also add the version of meilisearch (the engine)